docs/DatabaseCatalogAPI.yaml

```diff
@@ -677,6 +677,10 @@ components:
             For a list of valid codes [see here](https://unece.org/trade/uncefact/unlocode-country-subdivisions-iso-3166-2).
           type: string
           example: US
+        country:
+            description: The name of the country where the system is located.
```

Comment:
[suggestion]: Add a note to clarify that this field only supports English translations for now.

Reply:
this would apply to each field in the location -- should i specify it for each?

Reply:
You are right; except country_code, I think is a good idea to be explicit about it as the data in the catalog repo might be in a different language(with the exception of the feeds without datasets that they will have the same value)